modules/admin-top-bar/module.php

```diff
@@ -41,6 +41,8 @@ private function enqueue_scripts() {
 
 		wp_enqueue_style( 'elementor-admin-top-bar', $this->get_css_assets_url( 'admin-top-bar', null, 'default', true ), [], ELEMENTOR_VERSION );
 
```

Comment:
```suggestion

		/**
		 * Before admin top bar enqueue scripts.
		 *
		 * Fires before Elementor admin top bar  scripts are enqueued.
		 *
		 * @since 3.19.0
		 */
```